GitUI/CommandsDialogs/BrowseDialog/FormBrowseMenus.cs

```diff
@@ -195,8 +198,8 @@ private static void CreateToolStripSubMenus(ToolStrip senderToolStrip, ToolStrip
 
             return;
 
-            static void SaveVisibilitySetting(string key, bool visible) => AppSettings.SetBool(toolbarSettingsPrefix + key, visible ? null : false);
-            static bool LoadVisibilitySetting(string key) => AppSettings.GetBool(toolbarSettingsPrefix + key, true);
+            static void SaveVisibilitySetting(string key, bool visible) => AppSettings.SetBool(toolbarSettingsPrefix + key, visible);
```

Comment:
Why change this?

Reply:
valid question (I didn't notice this change)

Reply:
There was a choice that have been made, maybe to not "pollute" too much the setting file, to not store the setting value when the setting was 'true' (visible) but if the default value is 'false', this logic doesn't work anymore so you have to store the real value set by the user if it has been changed...

I had the same "problem" in another PR #10293

Reply:
The other possibility if we want to keep the current behavior is to pass the "default" value and compare with this default.
Do you prefer that?

Reply:
No strong preference, but I rather keep default as null, then it is easier to change without migrations.
No major difference here I guess. One or two lines of code or save defaults for all elements.

Reply:
Done. `null` is now for Default value.